src/features/pagination/test/pagination.spec.js

```diff
@@ -64,6 +64,8 @@ describe('ui.grid.pagination uiGridPaginationService', function () {
   describe('initialisation', function () {
     it('registers the API and methods', function () {
       expect(gridApi.pagination.getPage).toEqual(jasmine.any(Function));
+      expect(gridApi.pagination.getFirstRowIndex).toEqual(jasmine.any(Function));
```

Comment:
You should add more unit tests for these two functions than simply checking if they exist and are functions. Would you mind adding tests to ensure they return what is expected?